### PR TITLE
added failing test case + fix

### PR DESCRIPTION
**Pull request by Bl00D4NGEL**
Fixes https://github.com/rectorphp/rector-phpunit/issues

### CODE REVIEW

File: src/Rector/MethodCall/AssertIssetToSpecificMethodRector.php [hunk 1]
```diff
@@ -115,12 +137,13 @@ private function refactorPropertyFetchNode(Node $node, PropertyFetch $propertyFe
...
         $newArgs = $this->nodeFactory->createArgs([new String_($name), $propertyFetch->var]);
         $node->args = $this->appendArgs($newArgs, $oldArgs);
+        return $node;
     }
 
     /**
      * @param MethodCall|StaticCall $node
      */
-    private function refactorArrayDimFetchNode(Node $node, ArrayDimFetch $arrayDimFetch): void
+    private function refactorArrayDimFetchNode(Node $node, ArrayDimFetch $arrayDimFetch): ?Node
```
Comment (samsonasik) [inline src/Rector/MethodCall/AssertIssetToSpecificMethodRector.php:146]: nullable can be removed: 

```suggestion
    private function refactorArrayDimFetchNode(Node $node, ArrayDimFetch $arrayDimFetch): Node
```

**Comment by samsonasik** [comment]
Thank you, you can fix cs by run: 

```bash
composer check-cs -- --fix
```

**Review by samsonasik — APPROVED** [review]

**Comment by samsonasik** [comment]
I updated PR title with and desc to ref the issue and as it fix the issue.

**Comment by TomasVotruba** [comment]
Thank you :+1:

**Comment by samsonasik** [comment]
@TomasVotruba `rector/rector-phpunit` seems need new release for it?

**Comment by TomasVotruba** [comment]
@samsonasik  You're right :+1: Done - https://github.com/rectorphp/rector-phpunit/releases/tag/0.11.6

**Pull request merged**